src/main/java/database/helper/Settings.java

```diff
@@ -37,25 +42,19 @@ public void setPropertyFileName(String propertyFileName) {
         this.propertyFileName = propertyFileName;
     }
 
-    public Path getDatabasePath() {
-        if (propertyFileName == null) {
-            return defaultJsonDatabasePath;
-        }
+    public int getLimit() {
+        String limitStr = getProperty(LIMIT, String.valueOf(DEFAULT_LIMIT_VALUE));
+        return Integer.parseInt(limitStr);
+    }
 
-        var properties = new Properties();
-        var input = Settings.class.getClassLoader().getResourceAsStream(propertyFileName);
+    public int getOffset() {
```

Comment:
I think we don't need `offset` property at all. Please remove from code and `.properties` files.